mp/src/game/server/momentum/mom_system_saveloc.cpp

```diff
@@ -260,11 +260,49 @@ void CSaveLocSystem::LevelInitPreEntity()
     }
 }
 
+bool CSaveLocSystem::LoadStartMarks()
+{
+    if (m_pSavedLocsKV)
+    {
+        DevLog("Loaded startmarks from %s!\n", SAVELOC_FILE_NAME);
```

Comment:
This is nice but it doesn't mean the startmarks were loaded, since the rest of the function is actually doing the loading. This can move out to where `LoadStartMarks()` gets called, something like:

```
if (LoadStartMarks())
    DevLog("Loaded startmarks from %s!\n", SAVELOC_FILE_NAME);
```

Reply:
This has been moved to after the call to LoadStartMarks() in CMomentumPlayer::Spawn(), which better fits this log.